pkg/instrumentation/sdk.go

```diff
@@ -44,6 +45,8 @@ const (
 	sideCarName       = "opentelemetry-auto-instrumentation"
 )
 
+var volumeLimitSize = resource.MustParse("150Mi")
```

Comment:
The python images is the largest one ~100mb. I am using a larger number to get some buffer

Reply:
Should we make this configurable? I'm wondering what may happen for people building their own images that are larger than the otel defaults

Reply:
good point, I will make it configurable